frontend/templates/product-list/ProductListView.tsx

```diff
@@ -25,17 +30,28 @@ export function ProductListView({ productList }: ProductListViewProps) {
    const _ = useMenu({ attribute: 'facet_tags.Item Type' });
    const filters = computeProductListAlgoliaFilterPreset(productList);
 
+   const itemType = useDevicePartsItemType(productList);
+   const pagination = usePagination();
+   const page = pagination.currentRefinement + 1;
+   const productListModified = updateProductListWithItemTypeOverrides(
+      productList,
+      page,
+      itemType
+   );
+
    return (
       <>
          <SecondaryNavigation productList={productList} />
```

Comment:
Seems like it would make sense to use the same productListModified everywhere. It's hard to tell from here where it matters or not, but it seems like a good approach in general to be consistent.

Reply:
One idea would be to have ProductListView be a thin wrapper that does the item type overrides and then have the bulk of this be in another component that only has a single productList in scope.

Reply:
ok so because we use productList.title in the secondaryNavigation im not sure we should be passing it to that, it seems to make breadcrumbs wrong.